Modules/signalmodule.c

```diff
@@ -306,7 +307,7 @@ trip_signal(int sig_num)
 
     int fd;
 #ifdef MS_WINDOWS
-    fd = Py_SAFE_DOWNCAST(wakeup.fd, SOCKET_T, int);
+    fd = Py_SAFE_DOWNCAST(wakeup.fd, void *, int);
```

Comment:
Oh wait, we know this needs to be int sized? May as well just make it `int` throughout then (and safely cast when we store it).